static/fonts/fonts.css

```
@@ -13,3 +13,40 @@
 	font-style: italic;
 	font-display: swap;
 }
+
+/* Barlow */
+/* ExtraBold */
+@font-face {
+	font-family: 'Barlow';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Barlow" (font-family-name-quotes)

(font-family-name-quotes)